src/components/flat-table/flat-table.stories.mdx

```diff
@@ -198,6 +199,115 @@ Any cells preceding the `FlatTableRowHeader`s will automatically also be made st
   </Story>
 </Canvas>
 
+### Using row headers to create left and right hand side sticky columns
+
+In order to have right hand side sticky columns you should set the `FlatTableRowHeader` `stickyAlignment` prop to `right`.
+Any columns to following it will also be sticky.
+
+By default the left hand side columns have the prop set to `left`.
+It is important to make sure that the left hand side columns are not rendered before the right hand side ones.
```

Comment:
```suggestion
It is important to make sure that the left hand side columns are rendered before the right hand side ones.
```

Error while editing I assume? As the requirement is that the left columns *do* come before the right ones.

Reply:
good spot thanks 👍